EDA.qmd

```diff
@@ -379,17 +352,10 @@ ggplot(diamonds, aes(x = price)) +
   geom_freqpoly(aes(color = cut), binwidth = 500, linewidth = 0.75)
 ```
 
-The default appearance of `geom_freqpoly()` is not that useful for that sort of comparison because the height is given by the count and the overall counts of `cut` in differ so much, making it hard to see the differences in the shapes of their distributions:
+Note that ggplot2 uses an ordered color scale for `cut` because it's defined as an ordered factor variable in the data.
+You'll learn more about these in @sec-ordered-factors.
 
-```{r}
-#| fig-alt: >
-#|   Bar chart of cuts of diamonds showing large variability between the 
-#|   frequencies of various cuts. Fair diamonds have the lowest frequency, 
-#|   then Good, then Very Good, then Premium, and then Ideal.
-
-ggplot(diamonds, aes(x = cut)) + 
-  geom_bar()
-```
+The default appearance of `geom_freqpoly()` is not that useful for that sort of comparison because the height is given by the count and the overall counts of `cut` in differ so much, making it hard to see the differences in the shapes of their distributions.
```

Comment:
```suggestion
The default appearance of `geom_freqpoly()` is not that useful here because the height, determined by the overall count, differs so much across `cut`s, making it hard to see the differences in the shapes of their distributions.
```